src/Symfony/Bridge/Doctrine/Tests/Validator/Constraints/UniqueEntityValidatorTest.php

```diff
@@ -951,6 +951,40 @@ public function rewind(): void
     }
 
     public function testValidateDTOUniqueness()
+    {
+        $constraint = new UniqueEntity(
```

Comment:
The body of those tests is the same, only constraint is different. The [dataProvider](https://github.com/symfony/symfony/blob/7.1/src/Symfony/Bridge/Doctrine/Tests/Validator/Constraints/UniqueEntityValidatorTest.php#L441) should be used. This approach is already used  in the `UniqueEntityValidatorTest::provideConstraintsWithCustomRepositoryMethod`.

Reply:
I wrote the tests this way on purpose so that we don't have to rewrite them for Symfony 7.2 in #54744